src/components/HomePage.jsx

```diff
@@ -1,11 +1,35 @@
-import React from "react";
-import { Container } from "semantic-ui-react";
+import React, { useState, useEffect } from "react";
+import { Grid } from "semantic-ui-react";
+import { articlesIndex } from "../modules/apiHelper";
+import ArticleItem from "./ArticleItem";
 
 const HomePage = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    (async () => {
+      const articlesResponse = await articlesIndex();
+      setArticles(articlesResponse);
+    })();
```

Comment:
This looks fine to me but I would like you to check if React complains by issuing a warning. Check the console output in Cypress.